src/AppBundle/Service/Packagist/PackagistServiceProvider.php

```diff
@@ -144,7 +144,7 @@ private function excludeMaintainers(array $maintainers)
     private function calculateChecksum(array $fields = array()) {
         $string = '';
         foreach ($fields as $field) {
-            $string += $field;
+            $string .= $field;
```

Comment:
no, the first element (array key `updated`) of passed array `$fields` is an integer [link](https://github.com/ezsystems/ezplatform-com/pull/107/files#diff-3a86c44bc6c3f098e6aa6baa4fd45740L104), and the second one is a string. String cannot be added to integer - the operator used in this method should be concatenation `.`, not  addition `+`